config/locales/exceptions.en.yml

```diff
@@ -1685,7 +1691,7 @@ en:
         title: Duplicate Intake In Progress
         detail: Another duplicate intake is already in progress
         code: DR_duplicate_intake_in_progress
-        status: 422
+        status: 409
```

Comment:
This was an existing error response, but I don't see that any of the existing swagger or specs were changed accordingly - what gives? Were we not testing and/or swaggering it in the existing API's before this PR?

Reply:
It wasn't in the swagger docs prior to this PR. Would've been introduced when the service was initially integrated.